app/src/main/java/com/jerboa/ui/components/post/PostListingCard.kt

```diff
@@ -1006,7 +1018,7 @@ fun PostTitleBlock(
         (postUrl != null && VideoHostComposer.isVideo(postUrl))
 
     when {
-        videoPost && expandedImage -> {
+        videoPost && expandedImage && !disableVideoAutoplay -> {
             PostTitleAndVideoLink(
```

Comment:
Shouldn't this be internal to that composable? This makes it seem like you're also hiding the post title and video, rather than just not playing the video.

It should probably display a frame, and just not autoplay, if that's possible.